scss/_concept_problem.scss

```diff
@@ -602,8 +602,11 @@
     .ru-answer span {
         // display: block;
     }
+    .ru-answer a.eqn-preview {
+        font-size: 1.5rem !important;
+    }
     .ru-answer .katex span, .ru-answer .text-center span {
-        display: initial;
+        // display: initial;
```

Comment:
I'd love to know why @daviesian made it this way four years ago, but removing it really does seem to fix it!

Reply:
Yes, who knows?!